sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableAttributeBindingProvider.cs

```diff
@@ -33,8 +33,8 @@ private IBinding TryCreate(BindingProviderContext context)
                 return null;
             }
 
-            // ParameterBindingData, JArray, and ParameterBindingData[] are bound by separate binding methods using BindToInput<> specific to those respective types
-            if (parameter.ParameterType == typeof(JArray) || parameter.ParameterType == typeof(ParameterBindingData) || parameter.ParameterType == typeof(ParameterBindingData[]))
+            // ParameterBindingData and JArray are bound by separate binding methods using BindToInput<> specific to those respective types
+            if (parameter.ParameterType == typeof(JArray) || parameter.ParameterType == typeof(ParameterBindingData))
```

Comment:
Why are we removing the array? Don't we need that for collection binding?

Reply:
Matthew's comment here explains that we only need to support `ParameterBindingData` binding, and not `ParameterBindingData[]`. We're going to be doing the parsing on our end if we are doing collection binding

Reply:
Sounds good!